lib/dashboard.js

```diff
@@ -103,7 +103,7 @@ class Dashboard {
 	/**
 	 * Create a Bookshelf database connection.
 	 * @param {String} connectionString - A PostgreSQL connection string.
-	 * @returns {Bookshelf} A Bookshelf instance.
+	 * @returns {{VERSION: *}|{VERSION: *}} A Bookshelf instance.
```

Comment:
Is this a refactor gone wrong? Doesn't seem quite like a valid type...

Reply:
Gosh - yup... Not sure how that happened....?  Sounds like WebStorm just had a field day doing something silently - I don't remember changing this!